.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Documentation
```

Comment:
Pretty sure any changes to `.github/` will need to be made on their own and merged directly into master. The changes outside of `.github/` will go through the Copybara merge process in their own PR.

Reply:
Oh, didn't know that. Let me remove this one for now and have a second PR for it.

Reply:
Yeah this is a really unfortunate github "security" "feature" that integrations (e.g. Copybara) can't edit workflow files. There's an ongoing support ticket that's going around in circles :-/